--- libowfat-0.34-orig/trysendfile.c	2022-12-13 22:02:46.000000000 +0200
+++ libowfat-0.34/trysendfile.c	2026-07-31 11:55:28.935270148 +0300
@@ -63,7 +63,7 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <fcntl.h>
-#if defined(__GLIBC__) || defined(__dietlibc__)
+#if defined(__GLIBC__) || defined(__dietlibc__) || defined(__ANDROID__)
 #include <sys/sendfile.h>
 #else
 #include <linux/unistd.h>
