diff -u -r src-old/autosetup/sqlite-config.tcl src/autosetup/sqlite-config.tcl
--- src-old/autosetup/sqlite-config.tcl	2025-09-16 13:05:24.643784839 +0300
+++ src/autosetup/sqlite-config.tcl	2025-09-16 13:31:20.590786693 +0300
@@ -1881,6 +1881,7 @@
         [catch {exec $with_tclsh $::autosetup(libdir)/find_tclconfig.tcl} result] == 0} {
       set with_tcl $result
     }
+    set with_tcl $::env(TCLLIBDIR)
     if {"" ne $with_tcl && [file isdir $with_tcl]} {
       msg-result "$with_tclsh recommends the tclConfig.sh from $with_tcl"
     } else {
