--- src/generate-db.js
+++ src/generate-db.js
@@ -151,7 +151,7 @@
    var url;
    if (pkg_format === "debian")
      // Fetch the Contents.gz file for the given architecture from the apt mirror
-     url = `${repo.url}/dists/${repo.distribution}/Contents-${arch}.gz`;
+     url = `${repo.url}/dists/${repo.distribution}/Contents-${arch}`;
    else if (pkg_format === "pacman")
      // Fetch the ${repo}.files file for the given architecture from the pacman mirror
      url = `${repo.url}/${arch}/${repo.distribution}.files`;
@@ -158,7 +158,7 @@
    const response = await fetchURL(url);
  
    // Since we are using a gzip file, we need to decompress it
-   const data = await gunzipAsync(await response);
+   const data = await response;
    // Convert to string and split by new lines
    const lines = data.toString().split("\n");
  
