diff --git a/motds/motd.sh.in b/motds/motd.sh.in
index 1a76844..807f611 100644
--- a/motds/motd.sh.in
+++ b/motds/motd.sh.in
@@ -40,20 +40,11 @@ ${motd_indent}\e[1mInstall:\e[0m pkg install <package>
 ${motd_indent}\e[1mUpgrade:\e[0m pkg upgrade
 "
 
-
-if [ "$TERMUX_APP_PACKAGE_MANAGER" = "apt" ]; then
-
-    motd+="
-${motd_indent}\e[1mSubscribing to additional repos:\e[0m
-${motd_indent}\e[1mRoot:\e[0m pkg install root-repo
-${motd_indent}\e[1mX11:\e[0m  pkg install x11-repo
-
-${motd_indent}For fixing any repository issues,
-${motd_indent}try 'termux-change-repo' command.
+motd+="
+${motd_indent}To access shared storage on your device,
+${motd_indent}run termux-setup-storage command.
 "
 
-fi
-
 motd+="
 ${motd_indent}Report issues at \e[4mhttps://termux.dev/issues\e[0m
 "
diff --git a/motds/Makefile.am b/motds/Makefile.am
index 5e34b42..7c49c54 100644
--- a/motds/Makefile.am
+++ b/motds/Makefile.am
@@ -56,16 +56,9 @@ motd:
 	@echo " - Install: pkg install <package>" >> motd
 	@echo " - Upgrade: pkg upgrade" >> motd
 
-    ifeq ($(TERMUX_PACKAGE_MANAGER),apt)
-		@echo "" >> motd
-		@echo "Subscribing to additional repositories:" >> motd
-		@echo "" >> motd
-		@echo " - Root:    pkg install root-repo" >> motd
-		@echo " - X11:     pkg install x11-repo" >> motd
-		@echo "" >> motd
-		@echo "For fixing any repository issues," >> motd
-		@echo "try 'termux-change-repo' command." >> motd
-    endif
+	@echo "" >> motd
+	@echo "To access shared storage on your device," >> motd
+	@echo "run termux-setup-storage command." >> motd
 
 	@echo "" >> motd
 	@echo "Report issues at https://termux.dev/issues" >> motd
diff --git a/scripts/termux-setup-storage.in b/scripts/termux-setup-storage.in
index 51961ab..878d483 100644
--- a/scripts/termux-setup-storage.in
+++ b/scripts/termux-setup-storage.in
@@ -29,4 +29,5 @@ case "${TERMUX__USER_ID:-}" in ''|*[!0-9]*|0[0-9]*) TERMUX__USER_ID=0;; esac
 am broadcast --user "$TERMUX__USER_ID" \
 	-a "@TERMUX_APP_PACKAGE@.app.reload_style" \
 	--es "@TERMUX_APP_PACKAGE@.app.reload_style" "storage" \
+	--es "@TERMUX_APP_PACKAGE@.app.current_dir" "`pwd`" \
 	 "@TERMUX_APP_PACKAGE@" > /dev/null
